src/UI/Art/SpriteArtist/TerrainSpriteSet.java

```diff
@@ -262,7 +262,10 @@ private boolean isTerrainObject(Environment.Terrains terrainType)
       case CITY:
       case FACTORY:
       case FOREST:
+      case AIRPORT:
```

Comment:
Hmm... I really don't like having to hunt down all the switch cases to add enum values. We should make a note to re-evaluate how we do this so we have fewer updates when we add terrain types in the future. This is fine for now though.